examples-gallery/beginner/plot_wheel_on_bumpy_road.py

```diff
@@ -0,0 +1,370 @@
+r"""
+One Wheeler on Bumpy Road
```

Comment:
The model you are building here is traditionally referred to as a "quarter car model" not a "one wheeler". I think the former would be better for the title and language.